Calibration/HcalIsolatedTrackReco/src/EcalIsolatedParticleCandidateProducer.cc

```diff
@@ -211,8 +211,8 @@ EcalIsolatedParticleCandidateProducer::produce(edm::Event& iEvent, const edm::Ev
   // is put into the Event
   
 //  std::cout<<"put cand into event"<<std::endl;
-  std::auto_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
-  iEvent.put(pOut);
+  std::unique_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
+  iEvent.put(std::move(pOut));
 
```

Comment:
@wmtan 
'iptcCollection' is a raw pointer. I think this should be wrapped in a unique_ptr like you do in other places.